content/departments/engineering/design/templates/extra-large-template.md

```diff
@@ -0,0 +1,68 @@
+# Design Template: Extra Large
+
+- Estimate: More than a month
+- Notes:
+
+  - When creating the issue, use the label `design-estimated-XL`
+  - All phases for this project should involve a higher level of fidelity. Examples include additional test participants, multiple versions of designs, and more revision, and refinement than designs for smaller projects.
+  - Ideally, design should be given sufficient time in the problem space to explore and research.
+  - Design should be happening at least one iteration before engineering is planned. If user testing is required before or after design, this should be factored into the design time.
+  - These efforts require close collaboration with the Product Manager to scope and define the user problems being solved.
+
+✂️ -------------------------
+
+```markdown:
+## Design process
+
+  - Estimate: More than a month
+  - Weighting factors:
+    - Note: a project of this size requires resources that increase its risk profile.
```

Comment:
What does it mean to provide resources that increase a risk profile?

Reply:
More time and more people increase risk.